models/recurly__subscription_overview.sql

```diff
@@ -62,7 +62,7 @@ final as (
         {{ dbt_utils.datediff('subscription_enhanced.trial_started_at', 'subscription_enhanced.trial_ends_at', 'day') }} as trial_interval_days,
         subscription_enhanced.unit_amount
 
-        {{ fivetran_utils.persist_pass_through_columns('recurly_subscription_pass_through_columns', identifier='subscription_history') }},
+        {{ fivetran_utils.persist_pass_through_columns('recurly_subscription_pass_through_columns', identifier='subscription_enhanced') }},
```

Comment:
Same comment as above. Let me know if this makes sense.

Reply:
Yep!